### PR TITLE
Added Quarkus for Spring Developers blog post

### PR DESCRIPTION
**Pull request by jclingan**
A bit of due diligence in reviewing is recommended since this is my first PR to quarkus.io.

### CODE REVIEW

**Comment by jclingan** [comment]
Hoping @cescoffier can approve and merge. I should have added a reviewer.

File: _posts/2019-11-08-quarkus-for-spring-developers.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+== What existing knowledge can Spring developers bring to Quarkus?
+Quarkus’ Spring API compatibility includes https://quarkus.io/guides/spring-di-guide[Spring DI], https://quarkus.io/guides/spring-web-guide[Spring Web], and https://quarkus.io/guides/spring-data-jpa-guide[Spring Data JPA]. Additional Spring APIs are being planned like Spring Security and Spring Config. When running on the JVM, Quarkus applications can utilize virtually any Java library. Many libraries that are not dynamic in nature (do not use Java refection) will also compile to native. Lombok, a popular library among Spring developers for example, works with native compilation. To be clear, the Spring API support in Quarkus is not intended to be a complete Spring platform to re-host existing Spring applications. The intent is to offer enough Spring API support to make developing new applications with Quarkus a natural getting started experience. When combined with pre-optimized extensions, Quarkus delivers an https://quarkus.io/guides/[amazing amount of functionality] for microservices development. With all this being said, developers have successfully migrated Spring applications to Quarkus :-)
```
Comment (maxandersen) [inline _posts/2019-11-08-quarkus-for-spring-developers.adoc]: refection -> reflection

Comment (maxandersen) [reply _posts/2019-11-08-quarkus-for-spring-developers.adoc]: "Many libraries that are not dynamic in nature (do not use Java refection) will also compile to native" is a bit hard to parse imo, maybe do:
"Many libraries as long as they do not use Java reflection will compile to native" ..just a suggestion.

File: _posts/2019-11-08-quarkus-for-spring-developers.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+[IMPORTANT]
+====
+The Spring Framework is highly dynamic in nature. To address this, the Quarkus Spring compatibility extensions map Spring APIs to APIs in existing extensions that have already been optimized for fast startup, reduced memory utilization and native compilation, like RestEasy and CDI. Also, the Quarkus Spring compatibility extensions do not utilize the Spring application context. For these reasons, attempting to utilize additional Spring libraries will likely not work.
+====
+
+*Quarkus Spring Web Example*
+
+image::QuarkusSpringWeb.png[Quarkus Spring Web, 600]
```
Comment (maxandersen) [inline _posts/2019-11-08-quarkus-for-spring-developers.adoc]: instead of images you could just include the code like,

```
[source,java]
----
package ...

----
```

That would be nicer and copy/pasteable :)

File: _posts/2019-11-08-quarkus-for-spring-developers.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+image::QuarkusSpringRepository.png[Quarkus Spring Repository, 600]
+
+*Quarkus Spring Service + MicroProfile Fault Tolerance Example*
+
+image::QuarkusSpringService.png[Quarkus Spring Service Plus MicroProfile, 600]
+
+== Are there additional benefits for Spring developers?
+In addition to the improved memory utilization and startup time, Quarkus offers the following benefits to Spring developers:
+
+* *FaaS Runtime.* When compiled to a native binary, Quarkus applications can start in under .0015 seconds, making it possible to use the existing Spring and Java API knowledge with FaaS functions. (https://quarkus.io/guides/azure-functions-http[Azure], https://quarkus.io/guides/amazon-lambda[AWS Lambda])
```
Comment (maxandersen) [inline _posts/2019-11-08-quarkus-for-spring-developers.adoc]: Should expand "FaaS" at least once.  

"*FaaS Runtime*" -> "*Function-as-a-Service (Faas) Runtime*"

**Comment by maxandersen** [comment]
added a few notes but otherwise looks good rendered so that’s good ;)

File: _posts/2019-11-08-quarkus-for-spring-developers.adoc (hunk 2 above)
Comment (jclingan) [reply _posts/2019-11-08-quarkus-for-spring-developers.adoc]: Fixed. I was wondering how to do that :-)

File: _posts/2019-11-08-quarkus-for-spring-developers.adoc (hunk 1 above)
Comment (jclingan) [reply _posts/2019-11-08-quarkus-for-spring-developers.adoc]: Fixed and fixed.

File: _posts/2019-11-08-quarkus-for-spring-developers.adoc (hunk 3 above)
Comment (jclingan) [reply _posts/2019-11-08-quarkus-for-spring-developers.adoc]: Fixed.

**Comment by maxandersen** [comment]
looks good @jclingan - when do you prefer it to be pushed out ?

**Comment by jclingan** [comment]
@maxandersen Last Friday :-) Now is good.

**Comment by maxandersen** [comment]
rebased and updated date to today.
